rasa/core/utils.py

```diff
@@ -458,14 +458,15 @@ def handler(fut: Future) -> None:
     return handler
 
 
-def replace_floats_with_decimals(obj: Union[List, Dict]) -> Any:
+def replace_floats_with_decimals(obj: Union[List, Dict], round_digits: int = 9) -> Any:
```

Comment:
optional: What do you think about this name?
```suggestion
def replace_floats_with_decimals(obj: Union[List, Dict], decimal_precision: int = 9) -> Any:
```

Reply:
Haha - originally had that. I opted for digits as the round function uses [ndigits](https://docs.python.org/3/library/functions.html#round) as the second optional parameter descriptor.

Additionally, I didn't want to confuse with [getcontext().prec](https://docs.python.org/2/library/decimal.html).

Happy to change though 🙂

Reply:
makes sense, good for me then!